scripts/manage-db.py

```diff
@@ -84,13 +85,42 @@ def cleanup_releases_history(trans, dryrun=True):
 
     print "Total Deleted: %d" % total_deleted
 
+
+def extract_active_data(URL, loc="."):
```

Comment:
I see that "loc" is a directory that "dump.sql" will be written too. It would be better to not hardcode the filename, and let this be the filename to write to.